filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -1,3 +1,6 @@
+#if defined(_MSC_VER)
+    #include "./DriverAPI.inc.msvc"
```

Comment:
this one is a real shame; I'll see what I can do.

Reply:
Yes, I wish I could have made the new version work with msvc, but I'm no preprocessor wizard!

Reply:
Fixed now with this [PR](https://github.com/google/filament/pull/1686). Yeah!